Testscripts/Windows/STOR_VHDXResize_Basic.ps1

```diff
@@ -0,0 +1,222 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+<#
+.Synopsis
+    Verify basic VHDx Hard Disk resizing.
+.Description
+    This is a PowerShell test case script that implements Dynamic
+    Resizing of VHDX test case 2.3.1 and 2.3.2.
+    Ensures that the VM sees the newly attached VHDx Hard Disk
+    Creates partitions, filesytem, mounts partitions, sees if it can perform
+    Read/Write operations on the newly created partitions and deletes partitions
+    A typical test case definition for this test script would look
+    similar to the following:
+        <test>
+            <testName>VHDXResizeBasic</testName>
+            <testScript>SetupScripts\VHDxResizeToBigger.ps1</testScript>
+            <setupScript>SetupScripts\Add-VHDXHardDisk.ps1</setupScript>
+            <testparams>
+                <param>SCSI=0,0,Dynamic,512</param>
+                <param>NewSize=4GB</param>
+                <param>ControllerType=SCSI</param>
+                <param>Type=Dynamic</param>
+            </testparams>
+        </test>
+.Parameter testParams
+    Test data for this test case
+.Example
+    .\Testscripts\Windows\STOR_VHDXResize_Basic.ps1 -TestParams "SCSI=;NewSize=;"
+#>
+
+
+Param([String] $TestParams)
+
+Function Main
+{
+	$tps = Parse-TestParameters -XMLParams $CurrentTestData.TestParameters
+	$vmname = $allVMData.RoleName
+	$hvserver = $allVMData.HyperVHost
+    $ip = $AllVMData.PublicIP
+	$port = $AllVMData.SSHPort
+
+	try {
+
+		# if host build number lower than 9600, skip test
+		$BuildNumber = Get-HostBuildNumber $hvserver
+
+		if ($BuildNumber -eq 0) {
+			return $false
+		} elseif ($BuildNumber -lt 9600) {
+		    return "Skipped"
```

Comment:
As we are using try-Block, throw should be used. 
return "Skipped" is not permissable